code/game/objects/items/stacks/sheets/leather.dm

```diff
@@ -159,6 +159,12 @@ GLOBAL_LIST_INIT(xeno_recipes, list ( \
 	AddElement(/datum/element/dryable, /obj/item/stack/sheet/leather)
```

Comment:
add a grillable component that takes however long you want up here. makes more sense, right? you want it to dry when grilling, so let it use the grillable component.